maat-court-data-api/src/main/java/gov/uk/courtdata/dces/controller/DebtCollectionController.java

```diff
@@ -0,0 +1,41 @@
+package gov.uk.courtdata.dces.controller;
+
+import com.amazonaws.xray.spring.aop.XRayEnabled;
+import gov.uk.courtdata.dces.service.DebtCollectionService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.*;
+
+import java.time.LocalDate;
+import java.util.List;
+
+@RestController
+@RequestMapping("${api-endpoints.debt-collection-enforcement}")
```

Comment:
Are we expecting there to be a significant number of additional endpoints across multiple functional areas which will also fall under the URL path of `/api/internal/v1/debt-collection-enforcement` ?

The reason I ask, is that we should only be creating additional config items if it will be reused in other controllers, currently it is only being used by this controller.
Maybe replace
@RequestMapping("${api-endpoints.debt-collection-enforcement}")
with
@RequestMapping("${api-endpoints.assessments-domain}/debt-collection-enforcement")
or
@RequestMapping("/api/internal/v1/debt-collection-enforcement")
if appropriate?

Reply:
There won't be any more APIs (at least for now) and if that's the case, then we could look into refactoring that. 

Good shot on picking up the missing entry in the test package - apllication.yml